com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -145,10 +146,6 @@ private void testTf2(
           } else logger.warning(() -> "Encountered: " + pointerKey.getClass());
         });
 
-    // check the maps.
-    assertEquals(expectedNumberOfTensorParameters, methodSignatureToPointerKeys.size());
-    assertEquals(expectedNumberOfTensorParameters, methodSignatureToTensorVariables.size());
```

Comment:
Instead of removing these assertions, can we add the correct expected number? Please add a parameter called `expectedNumberOfTensorVariables` as a parameter.

Reply:
Added in bd73327e42f3ed256942703d278afe67e14b3dfe

Reply:
We don't need the commit number here, just the commit in the branch of this PR.

Reply:
I can see the change in the PR.